src/pudl/convert/datapkg_to_rst.py

```diff
@@ -0,0 +1,120 @@
+"""
+Module to convert json metadata into rst files.
+
+All of the information about the transformed pudl tables, namely their fields
+types and descriptions, resides in the datapackage metadata. This module makes
+that information available to users, without duplicating any data, by converting
+json metadata files into documentation-compatible rst files. The functions
+serve to extract the field names, field data types, and field descriptions of
+each pudl table and outputs them in a manner that automatically updates the
+read-the-docs.
+
+datapky2rst_many() reads the metadata json file, extracts the afformentioned
+data, and puts it into an rst file with tags and titles for each of the unique
+tables. This way, these tags can be referenced and appear in other rst files.
+
+"""
+
+import json
+import logging
+import sys
+from pathlib import Path
+
+from jinja2 import Template
```

Comment:
Don't forget to add `jinja2` to the `setup.py` -- since it's part of a script we're distributing and imported in the module here, it should go in the `install_requires` list of packages.